packages/ckeditor5-table/tests/tableproperties/tablepropertiesediting.js

```diff
@@ -227,6 +227,87 @@ describe( 'table properties', () => {
 					assertTRBLAttribute( table, 'borderColor', 'red' );
 					assertTRBLAttribute( table, 'borderStyle', 'solid' );
 					assertTRBLAttribute( table, 'borderWidth', '1px' );
+
+					// Also check the entire structure of the model.
+					// Previously the test was too loose in that regard.
+					expect( getModelData( editor.model ) ).to.equal(
+						'[<table ' +
+							'borderColor="{"top":"red","bottom":"red","right":"red","left":"red"}" ' +
+							'borderStyle="{"top":"solid","bottom":"solid","right":"solid","left":"solid"}" ' +
+							'borderWidth="{"top":"1px","bottom":"1px","right":"1px","left":"1px"}">' +
+							'<tableRow>' +
+								'<tableCell>' +
+									'<paragraph>' +
+										'parent:00' +
+									'</paragraph>' +
+								'</tableCell>' +
+								'<tableCell>' +
+									'<paragraph>' +
+										'child:00' +
+									'</paragraph>' +
+								'</tableCell>' +
+							'</tableRow>' +
+						'</table>]'
+					);
+				} );
+
+				// https://github.com/ckeditor/ckeditor5/issues/8393.
+				it( 'should not throw error when loading nested table with border styles', () => {
```

Comment:
It took some effort to come up with these input data. Don't you think that it would be good to use this occassion and also test whether what ends up in the model is correct too?

Reply:
Also, we prefer not to have one test covering 3 cases, so I'd prefer if you split this into 3 separate scenarios.

Reply:
I merged those cases based on the assumption we are just checking the exceptions. If I am going to check the models, I would also go with separating TC into separate tests.